simplified-ui-catalog/src/main/java/org/librarysimplified/ui/catalog/CatalogFeedViewModel.kt

```diff
@@ -621,7 +621,7 @@ class CatalogFeedViewModel(
     val pagedListConfig =
       PagedList.Config.Builder()
         .setEnablePlaceholders(true)
-        .setPageSize(50)
+        .setPageSize(100)
```

Comment:
Can't we keep 50 at a time?

Reply:
The backend determines how many titles it returns at a time, this doesn't change that. If this were set to 50, that could mean that the previous 50 results would be removed from memory when the next 50 are fetched. With this set to 100, the app still loads only 50 at a time, and the prefetch distance actually determines when the next page of results is loaded (when it gets to within 25 items from the end of the page, it fetches the next 50 titles).

These values are straight from the upstream app (like the earlier values were).